charter/index.html

```diff
@@ -167,40 +172,41 @@ <h1 id="title">PROPOSED Solid Working Group Charter</h1>
 
       <div id="background" class="background">
         <h2>Motivation and Background</h2>
-        <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created it.</p>
-
-        <p>Solid is largely based on existing open web standards, and extends them to give priority to individual and community autonomy. It allows them to keep authority over their identity, data, and privacy, and gives them the freedom to select applications and services that suit their needs.</p>
-
-        <p>Solid defines the concept of storage (one feature of the construct informally referred to as Solid Pods), in which users place data and control access to it, and a suite of protocols for applications to use storages.</p>
-
-        <p>Solid offers several advantages over more traditional architectures for data use by web services today, including:</p>
+        <p>It is common for web service providers to require users to surrender control over their own data, and store them in a way that is often restricted to certain systems, or can only be accessed by the applications that created them.
+        </p>
 
-        <ul>
-          <li>Storage is controlled by individuals or organizations, and can be delegated to other parties.</li>
-          <li>Interoperability between applications and various storage server implementations of the protocol.</li>
-          <li>A decentralized trust model enabling authentication and authorization mechanisms across the web.</li>
-        </ul>
+        <p><span class=new>A number of initiatives — such as <a href="https://indieweb.org/">IndieWeb</a>, <a href="https://www.w3.org/community/unhosted/">Unhosted</a>, and <a href="https://solidproject.org/">Solid</a> — have emerged to propose alternative models for web applications.
+        They are</span> largely based on existing open web standards, which they extend to give priority to individual and group autonomy.</p>
+        
+        <p>The <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> has been incubating <a href="https://solidproject.org/TR/">specifications</a> since 2018 towards that end, i.e., <q cite="https://www.w3.org/groups/cg/solid">to describe the interoperability between different classes of products by using web communication protocols, global identifiers, authentication and authorization mechanisms, data formats and shapes, notifications, and query interfaces.</q>
```

Comment:
@jyasskin — Do you have a recommendation for replacement markup? (I don't see any such in the issues you linked.) That will make it much easier to act on your comment.

Reply:
Just use quotation marks.
```suggestion
        <p>The <a href="https://www.w3.org/groups/cg/solid">W3C Solid Community Group</a> has been incubating <a href="https://solidproject.org/TR/">specifications</a> since 2018 towards that end, i.e., "to describe the interoperability between different classes of products by using web communication protocols, global identifiers, authentication and authorization mechanisms, data formats and shapes, notifications, and query interfaces."
```

The `cite` URL is no longer directly attached to the quotation, but that URL wasn't available to users outside of browser devtools anyway. The citation is still available through the link on "W3C Solid Community Group" at the beginning of the paragraph.

Reply:
Right, "quotation marks" is a near alternative, but there is nothing wrong with `<q cite` here either. `q` is still recomended in HTML5 for exactly this purpose and the `cite` attribute is used as per https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-q-element . Are there recent accessibility findings that recommends against the use of `q` or its deprecation in HTML5?

No objection to using just quotes... but prefer precision with `<q cite`.

Reply:
I share @csarven's taste for semantic markup, but I was convinced by @jyasskin's arguments about usability, discussed [here](https://github.com/whatwg/html/issues/10216).